### DISCUSSION TITLE
odrive_multi_interface died with code 11

### DISCUSSION THREAD

**Discussion by weixinchris** [Factor-Robotics/odrive_ros2_control#27] [Q&A]
![Screenshot from 2023-01-17 22-48-09](https://user-images.githubusercontent.com/78514694/213103312-db01e95d-5d69-45f5-9ff6-8e262062c29c.png)
I'm basically new to ROS2 and working on a Odrive based robotics project. Got an issue to run the first demo of odrive_multi_interface as shown in the picture. The process has died with an exit code 11. It seems like about the controller_manager, however don't know whether I missed some setup before running the demo program.
Any help will be appreciated.
Thanks,

**Comment by bakiiir** [comment]
u have no device plug in u must plug your odrive then run this launch file it wil be work normaly
